web/themes/custom/votegov/src/js/external-links.js

```diff
@@ -5,7 +5,8 @@
 (() => {
 
   // Enable external links behavior.
-  // `extlinkjs-ignore` attribute should be added to all icons in order to not render the uswds external links icon 
+  // `extlinkjs-ignore` attribute should be added to all icons in order to not render the uswds external links icon
+  //.filter( (externalLink) => !externalLink.matches()
```

Comment:
is this a comment that needed or maybe code left over?

Reply:
@clmedders Good catch, I will remove this comment

Reply:
comment was removed - see commit https://github.com/usagov/vote-gov-drupal/pull/784/commits/784c5fbdc32d242e2434e8b1aa1eed6666b55467